### PR TITLE
added conf_target and sat_per_byte to closeallchannels

### PR DESCRIPTION
**Pull request by michael1011**
Added the command line flags `conf_target` and `sat_per_byte` to the command `closeallchannels`.

[Was requested by joaoalmeida on Slack](https://lightningcommunity.slack.com/archives/C9RDCG8KT/p1522529600000079).

### CODE REVIEW

**Comment by halseth** [comment]
@wpaulino

File: cmd/lncli/commands.go [hunk 1]
```diff
@@ -789,7 +789,12 @@ var closeAllChannelsCommand = cli.Command{
 	By default, one is prompted for confirmation every time an inactive
 	channel is requested to be closed. To avoid this, one can set the
 	--force flag, which will only prompt for confirmation once for all
-	inactive channels and proceed to close them.`,
+	inactive channels and proceed to close them.
+
+	In the case of a cooperative closure, One can manually set the fee to
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: s/One/one

File: cmd/lncli/commands.go [hunk 2]
```diff
@@ -800,6 +805,18 @@ var closeAllChannelsCommand = cli.Command{
 			Usage: "ask for confirmation once before attempting " +
 				"to close existing channels",
 		},
+		cli.Int64Flag{
+			Name: "conf_target",
+			Usage: "(optional) the number of blocks that the " +
+				"transaction *should* confirm in, will be " +
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: Could you rephrase this to specify that these are the closing transactions of the channels being closed?

File: cmd/lncli/commands.go [hunk 3]
```diff
@@ -800,6 +805,18 @@ var closeAllChannelsCommand = cli.Command{
...
+				"used for fee estimation",
+		},
+		cli.Int64Flag{
+			Name: "sat_per_byte",
+			Usage: "(optional) a manual fee expressed in " +
+				"sat/byte that should be used when crafting " +
+				"the transaction",
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: Same comment as above.

File: cmd/lncli/commands.go [hunk 4]
```diff
@@ -935,6 +952,8 @@ func closeAllChannels(ctx *cli.Context) error {
 					OutputIndex: uint32(index),
 				},
 				Force: !channel.GetActive(),
+				TargetConf: int32(ctx.Int64("conf_target")),
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: Looks like we can just set the `conf_target` type to be `int32`.

Comment (michael1011) [reply cmd/lncli/commands.go]: There not way to get an `int32` value from `ctx`. It is done this way every time `conf_target` is used. @wpaulino

**Comment by michael1011** [comment]
Sorry for responding so late. I completely forgot about this pr :confused:

File: cmd/lncli/commands.go [hunk 5]
```diff
@@ -789,7 +789,12 @@ var closeAllChannelsCommand = cli.Command{
 	By default, one is prompted for confirmation every time an inactive
 	channel is requested to be closed. To avoid this, one can set the
 	--force flag, which will only prompt for confirmation once for all
-	inactive channels and proceed to close them.`,
+	inactive channels and proceed to close them.
+
+	In the case of cooperative closures, One can manually set the fee to
```
Comment (wpaulino) [inline cmd/lncli/commands.go]: Nit: s/One/one

**Comment by halseth** [comment]
Oldie but goodie. 

@michael1011 Can you give it a rebase? 😄

**Comment by michael1011** [comment]
Done @halseth
